react/components/UserDropdown/index.js

```diff
@@ -69,6 +70,8 @@ export default class UserDropdown extends Component {
 
           const { me } = data;
 
+          const recentlyJoined = moment().diff(moment(me.created_at), 'days') < 7;
```

Comment:
This is fine for now, but in general I want to get rid of moment (https://github.com/you-dont-need/You-Dont-Need-Momentjs). I think this might be the only place in js that's importing it?